### PR TITLE
Troubleshooting: expand and update

### PR DESCRIPTION
**Pull request by grahamperrin**
Add a section for Secure Boot.

Link to STABLE.

For hardware components, direct readers to hardware information that is provided by the FreeBSD Primary Release Engineering Team.

Add two dual-GPU scenarios.

Add a placeholder (ellipsis) subheading for the section that is incomplete.

Whilst here, reword things a little.

## Original (draft) Summary by Sourcery

Expand and update the troubleshooting guide by adding Secure Boot information, updating links, and rewording sections for clarity.

Documentation:
- Add information about disabling Secure Boot in BIOS settings for troubleshooting boot issues.
- Update the link to the FreeBSD STABLE branch for hardware support information.
- Include a placeholder for missing desktop environment issues in the troubleshooting guide.
- Reword and clarify sections on hardware support and installation media corruption.

### CODE REVIEW

**Comment by grahamperrin** [comment]
<https://docs.freebsd.org/en/books/faq/#hardware> in the rewritten book of FAQ  is not a good point of reference. 

Re: <https://codeberg.org/grahamperrin/freebsd-doc/pulls/21> the first FAQ link for graphics is to <https://wiki.freebsd.org/Graphics/Intel-GPU-Matrix>, which is explicitly stale with an obsolete table; and so on.

File: source/user/installation-guide/troubleshooting.md [hunk 1]
```diff
@@ -3,20 +3,37 @@ Troubleshooting
...
 
 ## The live system does not reach a graphical desktop environment.
 
-### The installation ISO may be outdated or require general bug fixes.
+…
+
+### The installation ISO may be outdated.
+
+Try creating a bootable USB using the [latest image](https://www.ghostbsd.org/download).
+GhostBSD is a rolling release and ocassionally updates and fixes are pushed to the latest build before a new official image is created.
```
Comment (grahamperrin) [inline source/user/installation-guide/troubleshooting.md:35]:
```suggestion
```

You might remove this sentence.

As things have modernised: will there _ever_ be more than a single version at <https://www.ghostbsd.org/download>? 

At the time of writing: 

- Official and Community images are both `24.10.1`
- there is no separate mention of a _latest_ (with a superior version).

Comment (ericbsd) [reply source/user/installation-guide/troubleshooting.md:49]: This part is outdated. We don't build the latest ISO like before.

File: source/user/installation-guide/troubleshooting.md [hunk 2]
```diff
@@ -3,20 +3,50 @@ Troubleshooting
...
+- computers that support [NVIDIA Optimus](https://en.wikipedia.org/wiki/Nvidia_Optimus)
+- some dual-GPU Macs.
+
+If your computer BIOS allows one of two GPUs to be disabled: please make this setting before your next attempt to boot the live system.
+
+Users of affected Apple hardware may find it impossible to set the Mac to use a single GPU.
+
+### The installation ISO may be outdated.
+
+Try creating a bootable USB using the [latest image](https://www.ghostbsd.org/download).
```
Comment (ericbsd) [inline source/user/installation-guide/troubleshooting.md:48]:
```suggestion
```

**Review by ericbsd — APPROVED** [review]

**Review by ericbsd — APPROVED** [review]
LGTM

**Comment by grahamperrin** [comment]
Thanks! 

Side notes: 

- the motivation for this PR was losing a potential user of GhostBSD (an existing user of Microsoft Windows) in FreeBSD Discord – at the tail end of one conversation, it was suspected that Secure Boot might be the simple explanation for the person metaphorically _getting nowhere_, after writing to a USB drive more than once
- the need to disable Secure Boot is not yet covered in FreeBSD installation information (e.g. <https://github.com/freebsd/freebsd-doc/blob/main/website/archetypes/release/installation.adoc>) or the documentation portal, there's discussion in Discord
- for <https://ghostbsd-documentation-portal.readthedocs.io/en/latest/user/installation-guide/troubleshooting.html#your-hardware-may-not-be-supported>, I was not _entirely_ happy about writing a need for the reader to mentally correlate (e.g. 1401502 and 14.1-STABLE) – IMHO it's not suitably newbie-friendly – but for now, it'll do.

**Pull request merged**